Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -135,11 +138,18 @@ public void periodic( )
 
       Pose2d llPose = lastResult.getBotPose2d_wpiBlue( );
 
-      if (lastResult.valid)
+      if (lastResult.valid && llPose.getX( ) != 0 && llPose.getY( ) != 0)
```

Comment:
Did we prove that we get a 0,0,0 pose with the valid setting as true? I don't remember the results.

Reply:
Yes valid is a boolean in Limelight Helpers and we still had the 0,0,0 problem with it true